tests/src/test/java/alluxio/security/MasterClientAuthenticationIntegrationTest.java

```diff
@@ -95,6 +97,26 @@ public void customAuthenticationDenyConnect() throws Exception {
     }
   }
 
+  @Test
+  @LocalAlluxioClusterResource.Config(
+      confParams = {PropertyKey.Name.SECURITY_AUTHENTICATION_TYPE, "SIMPLE"})
+  public void simpleAuthenticationIsolatedClassLoader() throws Exception {
```

Comment:
I am asking because I am wondering if it possible to assert on more explicit conditions? e.g., able to call `LoginUser.login()` with a different classloader?